course-template-bundle-spec.md

```diff
@@ -182,3 +183,16 @@ activity_options | ✓        | array       | `activity_options` is an array of
 -- id            | ✓        | string      | Reference to the unique identifier for the activity - `library/slug`.
 prompt           |          | dictionary  | Key is `locales` and each locale is a dictionary mapping locale codes to a prompt describing the step
 optional         |          | boolean     | `true` if the step is *not* required for completion
+
+### Retake Policies
+
+A retake policy defines any required cooldown periods, retake limits, and retake windows applicable to a given `CourseTemplate` activity.
+
+attribute               | required | type        | notes
+----------------------- | -------- | ----------- | -----------------------------------------
+id                      | ✓        | string      | A unique identifier for this retake policy
+activity_type           | ✓        | enum        | The type of retakeable activity to which this retake policy applies; one of `Lab` or `Quiz`
+activity_subtype        |          | string      | Distinguishes a subset of a given type of retakeable activity to which this retake policy applies (i.e. `graded` for quiz)
```

Comment:
nit:
```suggestion
activity_subtype        |          | string      | Distinguishes a subset of a given type of retakeable activity to which this retake policy applies (e.g. `graded` for quiz)
```

Reply:
also we should indicate that these subtypes can be any string and must match the subtypes given on the activities

Reply:
Done.